samples/Azure.Management.Storage/Generated/BlobServiceResource.cs

```diff
@@ -102,6 +102,7 @@ public virtual BlobContainerCollection GetBlobContainers()
         /// <param name="cancellationToken"> The cancellation token to use. </param>
         /// <exception cref="ArgumentException"> <paramref name="containerName"/> is an empty string, and was expected to be non-empty. </exception>
         /// <exception cref="ArgumentNullException"> <paramref name="containerName"/> is null. </exception>
+        [ForwardsClientCalls]
```

Comment:
besides solve the Get[Resource] test problem what's other impact if a method add this annotation

Reply:
It should not have other impact, actually the validation happens in test framework, so this method in our recently released beta packages can work as expected.